src/test/async_kill_with_threads_main_running.c

```diff
@@ -10,6 +10,8 @@ static void* start_thread(__attribute__((unused)) void* p) {
 int main(void) {
   pthread_t thread;
 
+  atomic_puts("ready");
```

Comment:
This test probably doesn't particularly care (you just can't put it in `start_thread`). If you prefer it after `pthread_create`, I'll move it.